### ISSUE TITLE
[NuGet.VisualStudioExtension] NuGetPackage crashes when solution config file ($(SolutionDir)\.nuget\NuGet.config) is read only

### ISSUE THREAD

**Issue by shiggan** [NuGet/Home#1103]
Windows 8.1, TFS 2010, Visual Studio 2015

I have a NuGet.config file in my solution ($(SolutionDir)\.nuget\NuGet.config).

When I open the solution; Visual Studio throws an error dialogue; and the following is logged in my ActivityLog.xml (%APPDATA%\Microsoft\VisualStudio\14.0\ActivityLog.xml)

```
... previous entries removed
  <entry>
    <record>1047</record>
    <time>2015/08/03 22:28:28.507</time>
    <type>Information</type>
    <source>VisualStudio</source>
    <description>Begin package load [NuGetPackage]</description>
    <guid>{5FCC8577-4FEB-4D04-AD72-D6C629B083CC}</guid>
  </entry>
  <entry>
    <record>1048</record>
    <time>2015/08/03 22:28:28.617</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [NuGetPackage]</description>
    <guid>{5FCC8577-4FEB-4D04-AD72-D6C629B083CC}</guid>
    <hr>80070005 - E_ACCESSDENIED</hr>
    <errorinfo>Access to the path 'D:\CODE\...\Development\IT6-Feature\1455\.nuget\NuGet.config' is denied.</errorinfo>
  </entry>
  <entry>
    <record>1049</record>
    <time>2015/08/03 22:28:28.618</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [NuGetPackage]</description>
    <guid>{5FCC8577-4FEB-4D04-AD72-D6C629B083CC}</guid>
    <hr>80070005 - E_ACCESSDENIED</hr>
    <errorinfo>Access to the path 'D:\CODE\...\Development\IT6-Feature\1455\.nuget\NuGet.config' is denied.</errorinfo>
  </entry>
</activity>
```

When I set $(SolutionDir)\.nuget\NuGet.config to be read/write; i no longer receive the error.

**Comment by shiggan** [comment]
.... sorry for noise; just saw the release notes :)

**Comment by emgarten** [comment]
This should be fixed in 3.1.1 which is available on the extension gallery.

**Comment by yishaigalatzer** [comment]
@shiggan if you still see it after the update feel free to re-open
